### ISSUE TITLE
Problem connecting to servers

### ISSUE THREAD

**Issue by alex384** [djtech42/YikYakTerminal#25]
How is YY banning users and how can you get around it? Changing device ID, IP address etc doesn't work

**Comment by deveyNull** [comment]
Yeah those things work, updates just broke everything in this program, API calls work different now.

**Comment by jakesylvestre** [comment]
Any chance we can update it and make it work again?
